packages/twenty-front/src/modules/activities/emails/components/EmailPreview.tsx

```diff
@@ -65,7 +78,7 @@ const StyledReceivedAt = styled.div`
 
 type EmailPreviewProps = {
   divider?: boolean;
-  email: Email;
+  email: TimelineThread;
```

Comment:
It's actually a thread then right?
```suggestion
  emailThread: TimelineThread;
```
OR simply
```suggestion
  thread: TimelineThread;
```